_data/accessibility-tests/table.yml

```diff
@@ -0,0 +1,110 @@
+      When using a keyboard to navigate a table,
+      a user can navigate using arrow keys or the tab key with no unexpected cues or instruction.
+    test_status: exception
+    status_details: |
+      The scrollable table does not scroll with keystrokes in JAWS.
+      A user would need to use their mouse to follow along with the focus indicator in JAWS.
+    github_issue_number: 5828
+    github_issue_repo: uswds
+    test_type: keyboard
+    version_tested: 3.6.0
+    wcag_criterion: 2.1.1
+  - summary: Table cells do not trap focus.
+    summary_additional: |
+      When using a screen reader and keyboard keystrokes to navigate a table,
+      you can easily navigate in and out of the table cells using only keystrokes.
+    test_status:
+    test_type: keyboard
+    version_tested:
+    wcag_criterion: 2.1.2
+  - summary: Focus indicator is clear on interactive items in tables.
+    summary_additional: |
+      When using a keyboard to navigate a table,
+      a focus indicator appears around all interactive elements (e.g. arrows for sorting, links).
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.5.0
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Table cells are read one at a time and provide context for each cell.
+    summary_additional: |
+      When using a screen reader to navigate a table,
+      the screen reader speaks one cell at a time and explains where each cell is in the table.
+      Tip: When using JAWS press the `ctrl` + `alt` keys to navigate within the table.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.0
+    wcag_criterion: 1.3.1
+  - summary: Table cells are read in a logical order.
+    summary_additional: |
+      When using a screen reader to navigate a table,
+      you hear the table cells for an entire row read in order from left to right before moving down to the next row.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.0
+    wcag_criterion: 1.3.2
+  - summary: Merged table headers are correctly associated with table cells.
+    summary_additional: |
+      When using a screen reader to navigate a table with merged header cells,
+      you can understand which lower-level data is organized under merged header cells.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.0
+    wcag_criterion: 2.1.1
+  - summary: The table has a header and it is associated with the correct table cells.
+    summary_additional: |
+      When using a screen reader to navigate a table,
+      you can clearly hear which column head the data falls under.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.0
+    wcag_criterion: 2.4.6
+  - summary: Screen readers correctly announce table elements.
+    summary_additional: |
+      When using a screen reader to navigate a table,
+      the screen reader announces the table and calls the header rows "column headers".
```

Comment:
> [!note]
> I did not incorporate this because we removed this test item in [e664553](https://github.com/uswds/uswds-site/pull/2538/commits/e66455340b2a0c2efae615d113213caf0b7645ad)